<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectManagementClass</class>
 <widget class="QWidget" name="ProjectManagementClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjectManagement</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 11pt &quot;Calibri&quot;;
}

/* QFrame */

#fr_todo, #fr_doing, #fr_done
{
background-color: rgba(236, 237, 239, 1);
border-radius:5px;
}

/* QPushButton */

#b_newtask_to_do, #b_newtask_doing, #b_newtask_done
{
	text-align:left;
	color: rgb(90, 90, 90);
}

/* QToolButton */

QToolButton#tb_todo, #tb_doing, #tb_done
{
background: rgba(178, 234, 255,0);
image:url(images/mainwindow/trois-points.png);
}

QToolButton#tb_todo:hover, #tb_doing:hover, #tb_done:hover
{
image:url(images/mainwindow/trois-points_hover.png);
}

/* QLabel */
#la_todo, #la_doing, #la_done
{
color : rgb(65,85,65);
font-weight:bold;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="vl_todo" stretch="3,1">
     <item>
      <widget class="QFrame" name="fr_todo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="hl_todo">
          <item>
           <widget class="QLabel" name="la_todo">
            <property name="text">
             <string>TO DO</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tb_todo">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vl_area_todo"/>
        </item>
        <item>
         <widget class="QPushButton" name="b_newtask_to_do">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Add new task</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>images/mainwindow/plus.png</normaloff>images/mainwindow/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vl_doing" stretch="3,1">
     <item>
      <widget class="QFrame" name="fr_doing">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,1,0">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="hl_doing">
          <item>
           <widget class="QLabel" name="la_doing">
            <property name="text">
             <string>DOING</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tb_doing">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vl_area_doing"/>
        </item>
        <item>
         <widget class="QPushButton" name="b_newtask_doing">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Add new task</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>images/mainwindow/plus.png</normaloff>images/mainwindow/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vl_done" stretch="3,1">
     <item>
      <widget class="QFrame" name="fr_done">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="hl_done">
          <item>
           <widget class="QLabel" name="la_done">
            <property name="text">
             <string>DONE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tb_done">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vl_area_done"/>
        </item>
        <item>
         <widget class="QPushButton" name="b_newtask_done">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Add new task</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>images/mainwindow/plus.png</normaloff>images/mainwindow/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
